skills/claude/quality-review/SKILL.md

```
@@ -352,15 +352,16 @@ Before transitioning, record the review verdict. The reviews value MUST be an ob
 ```
-exarchos_workflow({ action: "set", featureId: "<id>", updates: {
+exarchos_workflow({ action: "update", featureId: "<id>", updates: {
   reviews: { "quality-review": { status: "pass", summary: "...", issues: [] } }
-}, phase: "synthesize" })
+}})
+exarchos_workflow({ action: "transition", featureId: "<id>", target: "synthesize" })
 ```
 Then invoke `/exarchos:synthesize`.
 
 **NEEDS_FIXES:**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)